PS Script refactoring

juhlee-microsoft opened the pull request:
1. Replaced with PS standard cmdlet names
2. Block format
3. Replaced with 'Seconds' parameter

lizzha commented on LISAv2-Framework.psm1:
```diff
@@ -15,7 +16,7 @@ function Start-LISAv2 {
 
 		# [Required]
-		[ValidateSet('Azure','HyperV','OL', IgnoreCase = $false)]
+		[ValidateSet('Azure','HyperV','OL','WSL',IgnoreCase = $false)]
```
I will update the document

lizzha replied:
readme updated

lizzha commented on Libraries/TestLogs.psm1:
```diff
@@ -180,7 +180,7 @@ Function Get-SystemBasicLogs($AllVMData, $User, $Password, $currentTestData, $Cu
 		if ($TestPlatform -ne "OL") {
```
yes

lizzha commented on XML/GlobalConfigurations.xml:
```diff
@@ -53,4 +53,30 @@
             <HostFwdPort>1234</HostFwdPort>
         </TestCredentials>
     </OL>
+    <WSL>
```
In WSLController, it checks whether WSL is enabled on the host, and will abort if it's not enabled

lizzha replied:
comments added in the xml file, and readme updated for how to run test on WSL

ader1990 commented:
@lizzha can you add in the docs how to install/configure WSL on a Hyper-V node? thanks.

lizzha commented:
@ader1990 added in readme

ader1990 commented on Libraries/TestHelpers.psm1:
```diff
@@ -1061,4 +1061,43 @@ Function Remove-InvalidCharactersFromFileName
+			}
+		}
+	} else {
+		$ipAddresses = Get-NetIPAddress -AddressFamily IPv4 -AddressState Preferred -CimSession $ComputerName | `
```
here is a bit of duplicate code, but it can be refactored at a later time.

juhlee-microsoft committed:
PS Script refactoring

1. Replaced with PS standard cmdlet names
2. Block format
3. Replaced with 'Seconds' parameter

juhlee-microsoft committed:
Fixed with the comment

LiliDeng reviewed (approved):
Seems no logic change
